iOS/DuckDuckGo/DuckPlayer/DuckPlayer.swift

```diff
@@ -134,6 +134,13 @@ protocol DuckPlayerControlling: AnyObject {
     // Navigation Request Publisher to notify when DuckPlayer needs direct Youtube Nav
     var youtubeNavigationRequest: PassthroughSubject<URL, Never> { get }
     
+    /// Publisher that emits when Native DuckPlayer is dismissed
+    var playerDismissedPublisher: PassthroughSubject<Void, Never> { get }
+
+    /// The view and viewModel for the bottom sheet
+    var bottomSheetViewModel: DuckPlayerEntryPillViewModel? { get }
```

Comment:
The BottomSheet is a new UI component to present floating views that aren't native sheets.